### PR TITLE
 Build: Use a 6.0.200 SDK and macos workload P13

### PR DESCRIPTION
**Pull request by sandyarmstrong**

### CODE REVIEW

**Comment by sandyarmstrong** [comment]
We cannot actually upgrade to a newer macOS workload yet. Preview 11 is missing `NSKey` and newer previews (with `NSKey` restored) are not available in the public feeds.

**Comment by BretJohnson** [comment]
@sandyarmstrong - Thanks for starting this. When do you think vsmac will upgrade? I'm just trying to better understand the timeline & align.
Also cc @drasticactions, who just updated Hot Reload for this, here: https://github.com/xamarin/UITools/pull/650.

**Comment by sandyarmstrong** [comment]
I'm not sure. I just want to ease the eventual transition.

File: Xamarin.PropertyEditing.Mac/PropertyEditorPanel.cs [hunk 1]
```diff
@@ -26,7 +27,7 @@ public PropertyEditorPanel (IHostResourceProvider hostResources)
 		}
 
 		// Called when created from unmanaged code
-		public PropertyEditorPanel (IntPtr handle) : base (handle)
+		public PropertyEditorPanel (NativeHandle handle) : base (handle)
```
Comment (sandyarmstrong) [inline Xamarin.PropertyEditing.Mac/PropertyEditorPanel.cs:30]: These `NativeHandle` changes mean we cannot merge this until we're ready to build VSMac with a newer macos workload.

**Review by drasticactions — APPROVED** [review]

**Review by BretJohnson — APPROVED** [review]

**Pull request merged**